library/lib/build.js

```diff
@@ -204,6 +206,8 @@ module.exports = function build({ watch }) {
             watch && hotCssReplacementPlugin()
           ].filter(Boolean),
           web: [
+            commonChunksPlugin(),
```

Comment:
We discussed a new name for the `commonChunksPlugin`, something like `SharedModulesPlugin` ..?

Reply:
Sounds great